src/main/scala/cromwell/engine/backend/BackendCall.scala

```diff
@@ -54,21 +54,37 @@ trait ExecutionHandle {
   def result: ExecutionResult
 }
 
+sealed trait ReturnCode {
+  def code: Int
+
+  def asScriptReturnCode = this match {
+    case script: ScriptReturnCode => Option(code)
+    case _ => None
+  }
+
+  def asBackendReturnCode = this match {
+    case script: BackendReturnCode => Option(code)
+    case _ => None
+  }
```

Comment:
I asked myself the same question but BackendReturnCode is really more a BackendErrorCode.
At least the way we use it now is when the Call fails because of an error from the Backend (Preemption for example), we get the code but if it succeeds we only care about the Script code.
I guess we could still store the BackendReturnCode even on success though.